src/openforms/authentication/tests/factories.py

```diff
@@ -101,6 +101,7 @@ class Meta:
 
 class AttributeGroupFactory(factory.django.DjangoModelFactory):
     name = factory.Sequence(lambda n: f"attribute-group-{n}")
+    uuid = factory.Faker("uuid4")
```

Comment:
you don't need to specify a factory attribute when the model has `default=...` set - the model default will kick in.